src/reducers/initial.json

```diff
@@ -2,9 +2,11 @@
     "totalPatients": 0,
     "totalSimilarPatients": 0,
     "similarPatientTreatments": [],
-    "includedTreatments": ["radiation"],
+    "includedTreatments": [{"code":"385798007", "codeSystem":"http://snomed.info/sct", "displayName": "radiation therapy", "codeSystemName":"SNOMED"}],
```

Comment:
I'm not certain but I think @dehall deleted this file.  If not clear all of the values for the treatments, the values this file provides are also derived in the record to props functionality.

Reply:
I didn't delete the file itself, I think it's required for redux. I did clear out the values and just left an empty object so like you said the same thing should work here.